soc/xtensa/dc233c/linker.ld

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016 Cadence Design Systems, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Linker command/script file
+ *
+ * Linker script for the Xtensa platform.
+ */
+#include <xtensa-dc233c.ld>
```

Comment:
Irrelevant nitpick, but I don't see the value to putting the linker script in a separate file included from the official linker script file?  If it was shared with other SOCs or had some configurability, sure, but this seems like just creating a new file for no reason?

Reply:
I am just copying from `sample_controller` here. My plan is to see if we can use `dc233c` for `qemu_xtensa`, so we can get rid of `sample_controller`. At that point, I will need to clean all these up and refactor. That's going to take some time to unify some of the common linker script stuff for all Xtensa platforms.